From 9ae4f0f648ec564d0a882c42d53bf52b4ca7413f Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Fri, 15 Jan 2016 13:46:52 +0800
Subject: [PATCH 1/2] MLK-12265 rpmsg: imx: set the imx rpmsg tests be module

commit 2b83832fd551e05b71974766683056200ada3fe6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set the imx rpmsg tests to be module in defaut.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
(cherry picked from commit 20adce35ef6abfd6b0a52207febe069c7f09a1be)
Signed-off-by: Vipul Kumar <vipul_kumar@mentor.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/rpmsg/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/rpmsg/Kconfig b/drivers/rpmsg/Kconfig
index 8a19b5e2c3e1..9cae00463c9a 100644
--- a/drivers/rpmsg/Kconfig
+++ b/drivers/rpmsg/Kconfig
@@ -62,10 +62,12 @@ config HAVE_IMX_RPMSG
 
 config IMX_RPMSG_PINGPONG
 	tristate "IMX RPMSG pingpong driver -- loadable modules only"
+	default m
 	depends on RPMSG && m
 
 config IMX_RPMSG_TTY
 	tristate "IMX RPMSG tty driver -- loadable modules only"
+	default m
 	depends on RPMSG && m
 
 endmenu
-- 
2.17.1

